features/lorawan/LoRaWANStack.h

```diff
@@ -424,6 +442,25 @@ class LoRaWANStack: private mbed::NonCopyable<LoRaWANStack> {
      */
     uint16_t check_possible_tx_size(uint16_t size);
 
+private:
+
+    LoRaWANTimeHandler _lora_time;
+    LoRaMac _loramac;
+    LoRaPHY_region _lora_phy;
+    loramac_primitives_t LoRaMacPrimitives;
+
+    device_states_t _device_current_state;
+    lorawan_app_callbacks_t _callbacks;
+    radio_events_t *_mac_handlers;
+    lorawan_session_t _lw_session;
+    loramac_tx_message_t _tx_msg;
+    loramac_rx_message_t _rx_msg;
+    uint8_t _num_retry;
+    events::EventQueue *_queue;
+    bool _duty_cycle_on;
+    uint8_t _app_port;
```

Comment:
We can save some padding bytes by moving _app_port next to _num_retry

Reply:
Fixed